torch/csrc/jit/script/lexer.h

```diff
@@ -53,6 +53,8 @@ namespace script {
   _(TK_WHILE, "while", "while")                  \
   _(TK_EXPR_STMT, "expression statement", "")    \
   _(TK_RETURN, "return", "return")               \
+  _(TK_IS, "is", "is")                           \
+  _(TK_ISNOT, "is not", "is not")                \
```

Comment:
```
1 is                           not 2
```
works in python but fails here. My suggestion to is to fix it at the lexer level, similar to how TK_INDENT and TK_DEDENT are generated.